src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -426,6 +427,7 @@ public boolean isLegacyWatchApp() {
             resources.getResourceDirs(),
             resources.getDirsContainingResourceDirs(),
             resources.getResourceFiles()))) {
+      verifyResourceConflicts(resources, context.getSourcePathResolver());
```

Comment:
We should introduce this behaviour behind a config option as this has the potential to break existing apps which already include duplicate resources (which means we cannot update Buck without fixing any offending apps which is impractical).

The config for this should be off by default.

Reply:
done